Sources/StreamChat/ChatClient.swift

```diff
@@ -348,10 +348,12 @@ public class ChatClient {
         )
     }
     
-    /// Disconnects the chat client from the chat servers. No further updates from the servers
-    /// are received.
+    /// Stops active web-socket connection and cancels all queued API requests.
+    /// If there is a connection recovery flow running, it will be interrupted.
```

Comment:
I actually think the previous documentation was better. What the customer needs to know is that by calling `disconnect()` he will stop getting updates from the server. The changed documentation is describing on what happens internally. This documentation is good for `ConnectionController` but it should be in the public API.